src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -184,20 +172,39 @@ public void setRunType(String runType) {
     }
 
     /**
-     *
+     * @return exceptionsWaitTimeOut if user specified the exception time out in the config file otherwise
+     * return default webdriver time out
+     */
+    public int getExceptionsWaitTimeOut() {
+
+        return Optional.ofNullable(exceptionsWaitTimeOut)
+                .map(Integer::parseInt)
+                .orElse((int) webDriverWaitTimeout);
+    }
+
+    /**
+     * @param exceptionsWaitTimeOut set the timeout for exceptions which is using for tolerant methods
+     */
+    @JsonProperty("exceptionsWaitTimeOut")
+    public void setExceptionsWaitTimeOut(String exceptionsWaitTimeOut) {
+        this.exceptionsWaitTimeOut = exceptionsWaitTimeOut;
+    }
+
+    /**
      * @return Exceptions list that we will use to tolerate in tolerable action wrappers
      */
     public List<String> getExceptionsToHandleOnTolerantActions() {
         return exceptionsToHandleOnTolerantActions;
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes